src/components/TeamMemberCard/index.tsx

```diff
@@ -1,11 +1,69 @@
 import React from "react";
+import {
+  Card,
+  CardContent,
+  Typography,
+  Tooltip,
+  IconButton,
+} from "@mui/material";
+
+import GitHubIcon from "@mui/icons-material/GitHub";
+import EmailIcon from "@mui/icons-material/Email";
+import LinkedInIcon from "@mui/icons-material/LinkedIn";
 
 interface TeamMemberCardProps {
```

Comment:
I think this can be replaced by using the MemberInfo type from types/MemberInfo.ts. I'm still new to doing this, so I'm not 100% confident.

Reply:
i don't think the icons can be put in the memberInfo.ts - but I think it should be re-evaluated within the teammemberLinks component for the best method.

this comment did lead me to make some changes though - specifically in 17-19 in teamMemberCardFront - referencing the entire memberInfo as a passed prop instead of each one individually.